build.rs

```diff
@@ -178,19 +183,7 @@ fn cargo_build_bin(
         .join(std::env::var("PROFILE").unwrap())
         .join(bin_name);
 
-    // Strip the binary
-    let status = Command::new("strip")
-        .arg("--strip-unneeded")
-        .arg("-o")
-        .arg(&out_bin)
-        .arg(&target_bin)
-        .status()?;
-
-    // Failing that, just copy it into place
-    if !status.success() {
-        println!("cargo:warning=Failed to run `strip` on {:?}", target_bin);
-        std::fs::rename(&target_bin, &out_bin)?;
-    }
+    std::fs::rename(&target_bin, &out_bin)?;
```

Comment:
copy or rename? Would it mean it has to build the final binary every time again?

Reply:
Good catch, although it doesn't appear to take any significant amount of time to regenerate the final binary. I'll make it copy instead.